server/controllers/finance/cash.js

```diff
@@ -140,19 +137,20 @@ function detail(req, res, next) {
 
 /**
  * POST /cash
- * Creates a cash payment against several sales or a cautionary payment.  The
- * API also supports future offline functionality by either accepting a UUID or
- * generating it if it is not present.
+ * Creates a cash payment against one or many previous sales or a cautionary
+ * payment.  If a UUID is not provided, one is automatically generated.
  */
 function create(req, res, next) {
   'use strict';
 
-  var data = req.body.payment;
+  // alias insertion data
+  const data = req.body.payment;
```

Comment:
I followed the two links, and I noticed that : 

1. You are right ```const``` just means that the binding should not change.
2. Using const that way is a choice and not a rule, according to this portion of the article

> Everything I wrote here so far are facts. What follows is entirely subjective, but bear with me.
   Given the above, const makes code easier to read:
   within its scope, a const variable always   refers to the same object.
   With let there is no such  guarantee. 
  As a result, it makes sense to use let and const as   follows in your ES6 code:
 Use const by default
 only use let if rebinding is needed
 (var shouldn’t be used in ES6)

Here is the comment of @Andre Behrens

> Andre Behrens wrote on 5th February 2016 at 14:55: (reply)

   I would say if we don’t want to fool people into thinking const is for constants, then const was a poor syntactic choice.

   On our team, we treat it as a constant by convention, and everyone knows what that means. 
   Kind of like _quasiPrivate variables. For us, const means “I intend this to be constant”.
   It would be nice for the language not to have features that require this much explanation.

See also the second answer  [here](http://stackoverflow.com/questions/33295772/does-it-make-sense-to-use-const-for-all-variables-that-will-never-be-changed).